src/components/LandingPage/LandingPage.constants.ts

```diff
@@ -0,0 +1,7 @@
+export const LANDING_TEXTS = {
```

Comment:
Texts are hardcoded in English and capitalized oddly (e.g. 'Leet Code', 'LeetCode' appears elsewhere). The project requires using an i18n/localization solution and avoiding hardcoded user-visible strings in component constants.

**Sugerencia:**
```typescript
Move these strings to the i18n/locales system (e.g. use react-intl, i18next or project's i18n abstraction). Also normalize brand naming across code (decide if it is 'LeetCode' or 'Leet Code').
```